context-free.md

```diff
@@ -2,21 +2,55 @@
 
 ##  Who is this programming language for?
 
+This programming language is for people who want to be able to quickly make
+pictures that can easily be described as a context-free grammar. Many
+recursively generated pictures fall into this category, such as different kinds
+of fractals.
```

Comment:
Great answer and a completely different one from the one I gave. You answered the "what would someone be hoping to do with this language?" side of the question whereas I answered the "what kind of person would be attracted to this program if they knew of it?" side. I said that designers w/ an interest in tech or techies w/ an interest in art would be especially into CF art.